physionet-django/console/views.py

```diff
@@ -1072,6 +1073,14 @@ def user_management(request, username):
     Admin page for managing an individual user account.
     """
     user = get_object_or_404(User, username__iexact=username)
+    try:
+        aws_info = CloudInformation.objects.filter(user=user).values_list('aws_id', flat=True).get(pk=1)
```

Comment:
A simpler query is: 

`aws_info = CloudInformation.objects.get(user=user).aws_id`